src/Fixer/FunctionNotation/ReturnTypeDeclarationFixer.php

```diff
@@ -53,8 +103,31 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     /**
      * {@inheritdoc}
      */
-    protected function getDescription()
+    public function getDefinition()
     {
-        return 'There should be no space before colon and one space after it in return type declaration.';
+        $versionSpecification = new VersionSpecification(70000);
+
+        return new FixerDefinition(
+            'There should be one or no space before colon, and one space after it in return type declarations, according to configuration.',
+            array(
+                new VersionSpecificCodeSample(
+                    "<?php\nfunction foo() : string {};",
+                    $versionSpecification
+                ),
+                new VersionSpecificCodeSample(
+                    "<?php\nfunction foo() : string {};",
+                    $versionSpecification,
+                    array('space_before' => 'none')
+                ),
+                new VersionSpecificCodeSample(
+                    "<?php\nfunction foo(): string {};",
+                    $versionSpecification,
+                    array('space_before' => 'one')
+                ),
+            ),
+            'Rule is applied only in a PHP 7+ environment.',
+            "Configuration must have one element 'space_before' with value 'none' (default) or 'one'.",
+            array('space_before' => 'none')
```

Comment:
Probably should be done after #2366, when we know what the default configurations are, hehe.